tools.go

```diff
@@ -3,10 +3,6 @@
 package tools
 
 import (
-	// Generate deepcopy and conversion.
-	_ "sigs.k8s.io/controller-tools/cmd/controller-gen"
-	// Manipulate YAML.
-	_ "github.com/mikefarah/yq/v3"
 	// Generate embedded files.
 	_ "github.com/go-bindata/go-bindata/v3/go-bindata"
```

Comment:
Are we able to use go-get-tool for go-bindata as well?

Reply:
Surprisingly, this was a bit more complicated that I had anticipated due to how bloated the `manifests` target is right now, and exporting the PATH variable to point to this repositories' root `./bin` directory before running `go generate ...` wasn't working, and that command was always searching for packages in the vendor directory, despite no GOFLAGS being set in my local environment.

It's possible I'm missing something obvious, but given the o-f/api recent changes that contain massive changelogs due to dependency bumps, maybe we can tackle this in a follow-up, and/or generalize the value we see in the tools.go file going forward throughout the OLM-controller repositories?